std_msgs/Header header
string[] nodes
float64[] values